elastic/beats
# [Heartbeat] improve monitor state loader failure attempts

### Description
PR addresses couple of things on the state loader
+ We were waiting unnecessary for more than 4 seconds after all of the loading attempts.
+  Failure attempts were too frequent like the connection was reestablished in 0, 135 ms, 1200ms, 4300 ms, Now with the change, we will start from 0, 1200ms, 4300ms and exit with error code. 
+ Improve the logs around the last attempt and keep the message consistent. 

## How to test this PR locally

- Create a heartbeat browser/lightweight monitor
- Spin up a local ES 
- Close the local ES once the initial connection gets established
- Wait for the monitor to get finished, then check the attempts.
